front/src/common/Tabs.tsx

```diff
@@ -20,6 +20,7 @@ type TabsProps = {
   pills?: boolean;
   fullWidth?: boolean;
   fullHeight?: boolean;
+  className?: string;
```

Comment:
We probably don't need these changes anymore?

Reply:
Thank you